doc/man/zcash-cli.1

```diff
@@ -1,12 +1,12 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASH-CLI "1" "September 2017" "zcash-cli v1.0.12" "User Commands"
+.TH ZCASH-CLI "1" "November 2017" "zcash-cli v1.0.13-rc1" "User Commands"
 .SH NAME
-zcash-cli \- manual page for zcash-cli v1.0.12
+zcash-cli \- manual page for zcash-cli v1.0.13-rc1
 .SH DESCRIPTION
-Zcash RPC client version v1.0.12
+Zcash RPC client version v1.0.13\-rc1
 .PP
 In order to ensure you are adequately protecting your privacy when using Zcash,
-please see <https://z.cash/support/security/index.html>.
+please see <https://z.cash/support/security/>.
```

Comment:
Is this a manual change outside of `make-release.py`? I prefer to separate `make-release.py` changes from manual changes in the future, but for now this is acceptable. (If it's somehow the result of `make-release.py` that would also be notable and maybe a bug.)

Reply:
This is part of the `make-release.py` change - it updates man pages, which pull from changes to the help text in `init.cpp`.